core/domain/user_jobs_continuous.py

```diff
@@ -497,23 +486,26 @@ def get_dashboard_stats(cls, user_id):
         return {
             'total_plays': total_plays,
             'num_ratings': num_ratings,
-            'average_ratings': average_ratings
+            'average_ratings': average_ratings,
         }
 
 
 class UserStatsMRJobManager(
         jobs.BaseMapReduceJobManagerForContinuousComputations):
     """Job that calculates stats models for every user.
-    Includes: * average of average ratings of all explorations owned by the user
-              * sum of total plays of all explorations owned by the user
-              * impact score for all explorations contributed to by the user:
-    Impact of user is defined as S^(2/3) where S is the sum
-    over all explorations this user has contributed to of
-    value (per_user) * reach * fractional contribution
-    Value per user: average rating - 2
-    Reach: sum over all cards of count of answers given ^ (2/3)
-    Fractional contribution: percent of commits by this user
-    The final number will be rounded to the nearest integer.
+
+    Includes:
+        - Average of average ratings of all explorations owned by the user.
+        - Sum of total plays of all explorations owned by the user.
+        - Impact score for all explorations contributed to by the user.
+
+    The impact of user is defined as `S ** (2 / 3)` where S is the sum over all
```

Comment:
Let's keep the ^ instead of **, here and below. It's more standard mathematical notation.